### ISSUE TITLE
TypeError: only length-1 arrays can be converted to Python scalars

### ISSUE THREAD

**Issue by yesitsjess** [hart-lab/bagel#13]
Hi!

I have a log2FC file I've generated using a custom R script and the CRISPRcleanR package. It looks like this (but much bigger with non dummy data, obviously):
```
REAGENT_ID	GENE	SAMP1_BATCH1_REP1	SAMP1_BATCH1_REP2	
GENE1#1	GENE1	0.106184887170224	1.01519887444532
```
I'm giving the file as input to BAGEL like so:
`bagel/BAGEL.py bf -i input.tsv -o output.tsv -e bagel/CEGv2.txt -n bagel/NEGv1.txt -c 1,2`

I think the gene names are being capture correctly because I'm getting numbers under TrainEss, TrainNon and TestSet, but it fails on Iter 0 and I keep getting this error:
```
Traceback (most recent call last):
  File "bagel/BAGEL.py", line 505, in <module>
    xmax = round_to_hundredth( subx[f] )
  File "bagel/BAGEL.py", line 308, in round_to_hundredth
    return round( x*100) / 100.0
TypeError: only length-1 arrays can be converted to Python scalars
```

I thought it might be a rounding problem, as your tutorial shows hap1-t18.foldchange rounded to 3 decimal places, but if I round each log2FC using `trimws(format(round(x, 3), nsmall=3))` I still get the same error.

I would run `BAGEL.py fc` on the raw counts data but there's no option to turn off normalisation and I've already normalised them as part of the copy number correction step.

Can anyone suggest why I'm getting this error?

Many thanks,
Jess

**Comment by traverhart** [comment]
Hi Jess,

It looks like you're using the NEGv1 nonessentials genes list as both your
nonessential and essentials training sets - might that be the issue?

Cheers,
-Traver

---
Traver Hart, Ph.D.
Assistant Professor
Department of Bioinformatics and Computational Biology
Department of Cancer Biology
The University of Texas MD Anderson Cancer Center
http://www.hart-lab.net

**Comment by yesitsjess** [comment]
Ah good spot, that wasn't it though unfortunately, that was just a stupid mistake I made when trying to simplify for the purposes of this question. I'll edit that now so as not to be confusing. Thanks.

Update is that the only thing I've changed is updating from BAGEL build 009 to build 113 and now my error has changed:
`ValueError: array must not contain infs or NaNs`

Which is easier for me to understand so hopefully I can sort it myself now :)
